fstat/model.py

```diff
@@ -10,12 +10,25 @@
 )
 
 
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    username = db.Column(db.Integer, unique=True)
+    email = db.Column(db.String, unique=True)
+    profile_picture = db.Column(db.String)
+    token = db.Column(db.String)
+    name = db.Column(db.String)
+
+
 class Failure(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     signature = db.Column(db.String(1000), index=True)
     failures = db.relationship('FailureInstance',
                                backref='failure',
                                lazy='dynamic')
+    bugs = db.relationship('BugFailure', backref="failure")
```

Comment:
You'll need to generate a migration for this as well. I recommending waiting until we get the Github login branch merged and then rebasing this on top of master. That way you'll have less conflicts to deal with.